### ISSUE TITLE
[Feature Request] Make Gitlab resources importable

### ISSUE THREAD

**Issue by blaggacao** [gitlabhq/terraform-provider-gitlab#36]
### Affected Resource(s)
- gitlab_*

I'm a latecomer in the terraform ecosystem. I wonder, without having done a deep dive if it would be as simple as just implementin those lines:
https://github.com/terraform-providers/terraform-provider-fastly/blob/2dc6f92829de934b54ba7bdcf7f4f7a7aae6a9a2/fastly/resource_fastly_service_v1.go#L24-L26

Docs on resource importer:
https://github.com/hashicorp/terraform/blob/351c6bed79abbb40e461d3f7d49fe4cf20bced41/helper/schema/resource_importer.go

If would be really nice to have that feature. I'm unfortunately not really firm in go programming, else I'd have already done a PR.

**Comment by richardc** [comment]
I think that that was already done in #30 for `gitlab_project`, but reading #35 this seems like it was necessary but insufficient, so the fix in #35 is needed across all types.  I don't have a development environment configured to verify this right now, but I should have the bandwidth to look at this in the next few days.

**Comment by blaggacao** [comment]
Cool, sounds promising! Thank you!

**Comment by richardc** [comment]
In part this is a case of 'if it's not documented, it doesn't exist', as we had it in a form for project and group.

Here's what is left over from the existing resources:

- [x] `gitlab_project` (Documented and tweaked by #37)
- [ ] `gitlab_group` (Documented and tweaked by #38)
- [ ] `gitlab_deploy_key`
- [ ] `gitlab_label`
- [ ] `gitlab_project_hook`
- [ ] `gitlab_user`

**Comment by exarkun** [comment]
It looks like #39 was merged, although I'm not sure it works:
```
$ terraform import gitlab_user.xxx_xxx 14                                    
Acquiring state lock. This may take a few moments...                                                                                             
gitlab_user.xxx_xxx: Importing from ID "14"...
                                                        
Error: rpc error: code = Unavailable desc = transport is closing
          

Releasing state lock. This may take a few moments...
```

**Comment by timofurrer** [comment]
For the resources which return a secret / token only after creation we should implement the importer nonetheless, but omit that specific attribute only. See https://github.com/gitlabhq/terraform-provider-gitlab/pull/860 (or `gitlab_group_access_token` resource once the PR is merged :)
